adagrams/game.py

```diff
@@ -1,11 +1,132 @@
 def draw_letters():
-    pass
+    import random
+    RANGE_LOW = 0
+    RANGE_HIGH = 25
+
+    LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+    list_of_letters = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', "I", "J", "K", \
+                       "L", "M", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z"]
+    hand = []
+    while len(hand) < 10:
+        random_number = random.randint(RANGE_LOW, RANGE_HIGH)
+        letter = list_of_letters[random_number]
+        letter_value = LETTER_POOL.get(letter)
+
+        if letter in hand:
+            n = hand.count(letter)
+            if n < letter_value:
+                hand.append(letter)
+        else:
+            hand.append(letter)
+    print(hand)
+    return hand                  
+
 
 def uses_available_letters(word, letter_bank):
```

Comment:
Consider using a frequency map (aka dictionary of the letter frequency in the hand) similar to `LETTER_POOL`, so we can use its time complexity benefits and remove some of the nested loops in this function. Something like:
```python
for letter in word:
    # code here to check if the letter is in our frequency map and has a count larger than 0
    # if the letter exists, then decrement
    # otherwise we don't have a letter in our frequency map and we return False
    # if we get through the for loop, then we know all the letters were available and return True
```